lol-team/lol-team.js

```diff
@@ -220,7 +220,7 @@ importBtn.addEventListener('click', () => {
             }
         }
     });
-    for(let i=0; i<pArray.length; i++){
+    for(let i=0; i<pArray.length && pArray.length < 10; i++){
```

Comment:
![image](https://user-images.githubusercontent.com/55804872/222634406-eb953fcb-b9b3-4d3b-862e-2a30e5595818.png)

I'd use `numOfPlayers` instead hard coded number 10.
```suggestion
    for(let i=0; i<pArray.length && pArray.length < numOfPlayers; i++){
```